sheet.go

```diff
@@ -497,11 +497,27 @@ func (f *File) SetSheetBackground(sheet, picture string) error {
 		return ErrImgExt
 	}
 	file, _ := os.ReadFile(filepath.Clean(picture))
-	name := f.addMedia(file, ext)
+	return f.setSheetBackground(sheet, ext, file)
+}
+
+// SetSheetBackgroundFromBytes provides a function to set background picture
+// by given worksheet name and image data
+func (f *File) SetSheetBackgroundFromBytes(sheet string, picture []byte) error {
```

Comment:
Users can be specifying the image format by file `extension` name parameter:

```go
func (f *File) SetSheetBackgroundFromBytes(sheet, extension string, picture []byte) error
```

Reply:
Emmm, do you mean that I should use parameters to determine the data format instead of using the binary data?

Reply:
Yep, detection of image format by header bytes maybe not be accurate, the image payload binary data can be tampered with, this library cannot guarantee the image is valid even if it has correct header bytes, this is out of the scope of the library, we should leave this job to the user.

Reply:
Resolved